ui_framework/src/components/vertical_rhythm/_vertical_rhythm.scss

```diff
@@ -0,0 +1,23 @@
+.kuiVerticalRhythm {
```

Comment:
OK, I'll remove the mixins and components for now, and rename the terminology to `bottomMargin`.